android/app/src/main/java/com/example/myos/ui/ui/service/ServiceFragment.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.myos.ui.ui.service
```

Comment:
rename this package to `com.muh.myos.ui.ui.service`?
package `ui` inside another package `ui`?